DrivewayDentDeletion/Operators/deploy-im.sh

```diff
@@ -206,6 +221,76 @@ spec:
                 name: qm-${QM_NAME}-queues
                 items:
                   - myqm.mqsc
+  managedIntegrations:
+    list:
+    - kind: IntegrationRuntime
+      metadata:
+        name: ${IM_NAME}-ace-api
+      spec:
+        template:
+          spec:
+            containers:
+              - name: runtime
+                resources:
+                  requests:
+                    cpu: 300m
+                    memory: 368Mi
+        logFormat: basic
+        barURL: ${API_URL}
+        configurations: ${CONFIGURATIONS}
+        version: '12.0'
+        replicas: 1
```

Comment:
```suggestion
```
I think these are not needed as set by IA?

Reply:
replicas yes though in default the version we apply is `"12.0.7.0-r2"` if we are happy with that then it can be taken out

Reply:
Did we implement so if the IA version is a channel it uses a channel for IR? Ideally the only version should be for the IA.